src/Authentication/Controllers/RequestSystemUserController.cs

```diff
@@ -91,6 +96,13 @@ public async Task<ActionResult<RequestSystemResponse>> CreateRequest([FromBody]
             SystemId = createRequest.SystemId,
         };
 
+        SystemUser? existing = await _systemUserRepository.GetSystemUserByExternalRequestId(externalRequestId);
+        if (existing is not null)
```

Comment:
And can we add this to the validationerror list, so is consistent across the system user implementation

Reply:
ok, good idea